### DISCUSSION TITLE
Tokens sent on Ethereum haven't arrived to Solana

### DISCUSSION THREAD

**Discussion by GiraffeTheWizard** [wormhole-foundation/wormhole#319] [Problems / bugs] [answered]
Hello,

I tried to bridge ibBTC from Ethereum to Solana, but the funds I sent never appeared in my wallet on Solana. 
I used wormholebridge.com and Phantom wallet.

Here's the tx: https://explorer.solana.com/tx/UEcC13bYRrK4MLdjP4Ykkjiy1rMP15GvjSQazDE7niVJNodaj6ibesUSVCCZcgpa1LgdNCHYi16wYvD7Ap1YaDs

**Comment by leoluk** [comment]
Please provide the Ethereum transaction hash.

**Reply by GiraffeTheWizard** [reply]
here:
https://etherscan.io/tx/0xe497c578519ce5c531852a0b5a3b0075bcf8c3c33bdc59a4d9903c9341124399

**Comment by leoluk** [comment, marked as answer]
The Solana transaction had to be retried manually, check if your tokens arrived now: https://explorer.solana.com/tx/Stj2U4ohSXg87sRGp5ETCWXXQ5A9TxX7gNxbAPjHEjC82cASTokdk5bdr4FprFDyCqUHdWweLaM1xUUB4tLSCNA

---

Digest: 0e2b4ec73f9fee5c10d1f69b2d5384a5b60edb23d3af385d4d5d063af581e957

**Reply by GiraffeTheWizard** [reply]
the tokens have arrived, thank you!
